docs/changelog.rst

```diff
@@ -496,20 +499,20 @@ v3.0.0a3 (2021-06-25)
 
 * Pagination has changed significantly. (:pr:`418`)
 
-** Methods which support pagination like ``TransferClient.endpoint_search`` no
-   longer return an iterable ``PaginatedResource`` type. Instead, these client
-   methods return ``GlobusHTTPResponse`` objects with a single page of results.
+  * Methods which support pagination like ``TransferClient.endpoint_search`` no
+    longer return an iterable ``PaginatedResource`` type. Instead, these client
+    methods return ``GlobusHTTPResponse`` objects with a single page of results.
 
-** Paginated variants of these methods are available by renaming a call from
-   ``client.<method>`` to ``client.paginated.<method>``. So, for example, a
-   ``TransferClient`` now supports ``client.paginated.endpoint_search()``.
-   The arguments to this function are the same as the original method.
+  * Paginated variants of these methods are available by renaming a call from
+    ``client.<method>`` to ``client.paginated.<method>``. So, for example, a
+    ``TransferClient`` now supports ``client.paginated.endpoint_search()``.
+    The arguments to this function are the same as the original method.
 
-** ``client.paginated.<method>`` calls return ``Paginator`` objects, which
-   support two types of iteration: by ``pages()`` and by ``items()``. To
-   replicate the same behavior as SDK v1.x and v2.x ``PaginatedResource``
-   types, use ``items()``, as in
-   ``client.paginated.endpoint_search("query").items()``
+  * ``client.paginated.<method>`` calls return ``Paginator`` objects, which
+    support two types of iteration: by ``pages()`` and by ``items()``. To
+    replicate the same behavior as SDK v1.x and v2.x ``PaginatedResource``
+    types, use ``items()``, as in
+    ``client.paginated.endpoint_search("query").items()``
```

Comment:
So this doesn't actually render in generated docs as intended. You can see examples in our [public rendered changelog](https://globus-sdk-python.readthedocs.io/en/stable/changelog.html) under v3.12.0 & v3.11.0

I made the change here specifically because I wanted to render docs locally (using `make docs`) to ensure my generated docs were up-to-snuff & `make docs` was failing for local runs (maybe I have default configuration set to fail on warnings or something) as a result of the tool noticing "looks like you're trying to nest some bullet points, can I help with that?"

With that being said, I don't plan on doing any more local doc rendering for this change, so I can revert & put this out in a new PR if you'd like a cleaner git log.

Reply:
I had missed the earlier note

> Reformatted some broken nested bullet points in the existing changelog

(which you have now struck-through).

If I had understood correctly that there was an _error_ with the changelog, rather than this being a different way of writing the same thing, I probably wouldn't have asked you to revert. That's my mistake; sorry.

I hadn't realized there was an honest-to-goodness formatting error here; partly because markdown formats are so... "forgiving". Let's proceed here without this change in the current PR, but I would welcome a separate PR for this very enthusiastically.

Reply:
It's an error in the lightest possible interpretation of the word. The changelog renders two asterisks instead of a nested bullet point at a couple places.

I'm also a proponent of decoupling bugfixes (however minor they may be) from feature additions to improve the tracability & stateful iteration of development in repos. So I don't mind practice what you preach callout.